metis-normalization/src/main/java/eu/europeana/normalization/dates/edtf/EdtfDatePart.java

```diff
@@ -18,6 +18,7 @@ public class EdtfDatePart implements Serializable {
   private boolean uncertain;
   private boolean approximate;
   private boolean unknown;
+
   // TODO: 25/07/2022 What is unspecified? It's not in the documentation, is it the same as unknown?
```

Comment:
It relates to edtf format. This case is if there is an edtf formatted date already in the document and we are trying to parse it. This field will indicate if that date is an "unspecified" one.

Examples:
`1900/` (unspecified end)
`1900/?` (unknown)